jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.vue

```diff
@@ -36,8 +34,9 @@
     <v-row>
       <v-text-field
         ref="n_moment"
+        type="number"
         label="Moment"
-        v-model="n_moment"
+        v-model.number="n_moment"
```

Comment:
fixed this while I was here... using `.number` along with `IntHandleEmpty` removes the need for error handling in the plugin.